internal/renderer/render.go

```
@@ -0,0 +1,259 @@
+				iw, ih := w*3/5, hpx*3/5
+				x0, y0 := (w-iw)/2, (hpx-ih)/2
+				draw.Draw(img, image.Rect(x0, y0, x0+iw, y0+ih), &image.Uniform{c}, image.Point{}, draw.Src)
+			}
+		}
+		path := filepath.Join(dir, fmt.Sprintf("frame_%06d.png", f))
+		fh, err := os.Create(path)
+		if err != nil {
+			return f, fmt.Errorf("renderer: 建帧文件 %s: %w", path, err)
+		}
+		if err := png.Encode(fh, img); err != nil {
+			fh.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `fh.Close` is not checked

(errcheck)